lib/ui/profitloss/RecordTransactionListItem.dart

```diff
@@ -27,7 +27,7 @@ class _LocalisedStrings {
 }
 
 class _Strings {
-  static final emptyString = "";
+  static final emptyString = Intl.message("");
```

Comment:
I think that we don't need to localise an empty string :)
```suggestion
  static final emptyString = '';
```

Reply:
👍